hack/remove-members.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var repoRoot string
+
+func main() {
+	flag.StringVar(&repoRoot, "root", ".", "Root of the repository")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not")
+
+	flag.Parse()
+
+	if len(flag.Args()) < 1 {
+		fmt.Print("Usage: remove-members [flags] <file-containing-members-list>\n\nFlags:\n\t-root\tstring\tpath to root directory of the repository\n\t-dryrun\tboolean\tEnable/Disable dryrun (Default: Enabled)")
```

Comment:
Would be better to use `flag.Usage` - https://golang.org/pkg/flag/#pkg-variables

Reply:
Updated Usage
```
Usage: remove-members [flags] <file-containing-members-list>
  -dryrun
        Enable Dryrun or not. This flag controls whether the changes are simulated or live. If the changes are simulated it will only print the removal details and the commit message (default true)
  -path string
        Path to config directory/subdirectory (default ".")
```